templates/bundle/Dockerfile

```diff
@@ -68,7 +68,7 @@ HEALTHCHECK \
 	--interval=4s \
 	--start-period=10s \
 	--timeout=1m \
-	CMD curl -fsS "http://localhost:8080/c/portal/layout" || exit 1
```

Comment:
@kiwm, just including the parameters `--cookie` and `--cookie-jar` here wouldn't be enough to solve the problem?